src/exchange/billing/account-renew/billing-account-renew.html

```diff
@@ -0,0 +1,136 @@
+                    </oui-radio>
+                </oui-column>
+                <oui-column
+                    data-title="::'exchange_update_billing_header_terminate' | translate">
+                    <oui-radio name="renewPeriod-{{::$rowIndex}}"
+                        data-ng-if="!$ctrl.account.partial && $ctrl.canBeDeletedAtExpiration()"
+                        data-model="$row.renewPeriod"
+                        data-on-change="$ctrl.trackSelected($row.primaryEmailAddress, $row.renewPeriod)"
+                        data-value="'DELETE_AT_EXPIRATION'">
+                        {{::'exchange_update_billing_periode_value_DELETE_AT_EXPIRATION' | translate}}
+                    </oui-radio>
+                </oui-column>
+            </oui-datagrid>
+
+            <oui-message class="mb-5"
+                data-type="warning"
+                data-ng-if="$ctrl.canBeDeletedAtExpiration() && $ctrl.services.exchangeVersion.isAfter(2010, $ctrl.exchange) && !$ctrl.account.partial && $ctrl.model.displayDeleteWarning">
+                <span data-translate="exchange_update_billing_periode_delete_warning"></span>
+            </oui-message>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{::'exchange_update_billing_validation_title' | translate}}"
+            data-description="{{::'exchange_update_billing_periode_resume' | translate}}"
+            data-loading="$ctrl.submitLoader">
+            <div class="oui-datagrid-responsive-container" data-ng-if="!$ctrl.submitLoader">
+                <div class="oui-datagrid-responsive-container__sticky-container">
+                    <div class="oui-datagrid-responsive-container__overflow-container">
+                        <table class="oui-datagrid">
+                            <thead class="oui-datagrid__headers">
+                                <tr>
+                                    <th scope="col" data-translate="exchange_tab_ACCOUNTS_table_email" class="oui-datagrid__header"></th>
+                                    <th class="text-center oui-datagrid__header" scope="col" data-translate="exchange_update_billing_header_new_periode"></th>
+                                </tr>
+                            </thead>
+
+                            <tbody>
+                                <tr class="oui-datagrid__row" data-ng-repeat="account in ($ctrl.bufferedAccounts.list.results | orderBy:'primaryEmailAddress':false) track by $index">
+                                    <td data-ng-bind="account.primaryEmailDisplayName" class="oui-datagrid__cell"></td>
+                                    <td class="text-center oui-datagrid__cell" data-ng-bind="('exchange_update_billing_periode_value_' + account.renewPeriod) | translate"></td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+
+            <div class="alert alert-warning"
```

Comment:
A `oui-message` would be better

Reply:
Yes, I'll change it. Came from the previous template.